superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu

```diff
@@ -810,13 +808,16 @@ int main(int argc, char **argv) {
                     args_list.push_back(args);
                 }
             }
+            if (args.numa_id != 0) {
```

Comment:
For GPU PCIe version, is `Remove NUMA binding for device-to-device tests because NUMA doesn't affect performance;` correct? Can we have a option for this?

Reply:
In P2P tests, memory is allocated on GPU, and all transfers are not related to NUMA node. I think CUDA official p2pBandwithLatencyTest also doesn’t differentiate NUMA nodes.

Reply:
Sure.